.github/workflows/k8s-generated.yaml

```diff
@@ -0,0 +1,29 @@
+name: K8S generated file check
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+jobs:
+  generate-k8s-api:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.2
+      - name: Checkout code
+        uses: actions/checkout@v1
```

Comment:
We've tried to use `v2` in other GitHub Actions.

Reply:
oh let me bump this as well, this was copied from my old configuration.